examples/stm32/nucleo-h743zi-baremetal/Makefile

```diff
@@ -25,11 +25,11 @@ flash: firmware.bin
 
 # Requires env variable VCON_API_KEY set
 DEVICE_URL ?= https://dash.vcon.io/api/v3/devices/6
-test: update
-	curl --fail-with-body -su :$(VCON_API_KEY) $(DEVICE_URL)/tx?t=5 | tee /tmp/output.txt
-	grep 'ticks:' /tmp/output.txt
-#	grep 'Ethernet: up' /tmp/output.txt
-#	grep 'MQTT connected' /tmp/output.txt
+test:	EXTRA_CFLAGS += -DUART_DEBUG=UART1
+test:	update
+	curl --fail -su :$(VCON_API_KEY) $(DEVICE_URL)/tx?t=5 | tee /tmp/output.txt
```

Comment:
Nit: use `--fail-with-body` as it gives a clue about the failure

Reply:
done
(my version does not support either form...)